### PR TITLE
Added buttons provisioning autohotkey script

### PR DESCRIPTION
**Pull request by BraveKerry**
Updated the documentation here: 
https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-6207

### CODE REVIEW

File: ops/rak_button_firmware.ahk [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+SendInput AC1F09FFF9157201
+Sleep, 1000
+SendInput {Tab}{Right}
+SendInput AC1F09FFF9157201
+Sleep, 1000
+SendInput {Tab}
+SendInput AC1F09FFF9157201
+Sleep, 1000
+SendInput {Tab}
+SendInput AC1F09FFF9157201
```
Comment (trisapeace) [inline ops/rak_button_firmware.ahk:156]: So this will only work for production, not for dev, right?

File: ops/rak_button_firmware.ahk [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+if ErrorLevel
+    MsgBox, CANCEL was pressed.
+else
+    MsgBox, You entered "%name%"
+InputBox, counter, Start Counter, the first counter number (i.e. 1), , 640, 480
+if ErrorLevel
+    MsgBox, CANCEL was pressed.
+else
+    MsgBox, You entered "%counter%, buttons will be named starting from %name%_%counter%"
+Return
```
Comment (trisapeace) [inline ops/rak_button_firmware.ahk:177]: Are your instructions (https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-6207) out of date? It looks like you've creating a GUI to get the locationid and counter.

File: ops/rak_button_firmware.ahk [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+SendInput at{+}get_config=lora:status
+Sleep, 500
+PressEnter()
+Sleep, 500
+PressEui()
+PressEui()
+SendInput ^c
+Sleep, 500
+
+; google sheets paste eui
```
Comment (trisapeace) [inline ops/rak_button_firmware.ahk:79]: How does it know which Google Sheet to use?

File: ops/rak_button_firmware.ahk [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+Sleep, 500
+
+; check interval change
+WinActivate, ahk_exe RAK_SERIAL_PORT_TOOL_V1.2.1.exe
+Sleep, 500
+PressTxtBox()
+Sleep, 500
+Clear()
+Sleep, 500
+SendInput at{+}get_config=device:status
```
Comment (trisapeace) [inline ops/rak_button_firmware.ahk:95]: At this point, are you looking at the values on the screen and checking them?

File: ops/rak_button_firmware.ahk [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+Sleep, 500
+PressEnter()
+Sleep, 500
+
+; check status
+PressTxtBox()
+Sleep, 500
+Clear()
+Sleep, 500
+SendInput at{+}get_config=lora:status
```
Comment (trisapeace) [inline ops/rak_button_firmware.ahk:70]: At this point, are you looking at the values on the screen and checking them?

File: ops/rak_button_firmware.ahk [hunk 6]
```diff
@@ -0,0 +1,184 @@
+; Created 2022-07
+; by Peter Scholtens and Kerry Wang
+; Instructions can be found here: https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-6207
```
Comment (trisapeace) [inline ops/rak_button_firmware.ahk:3]: I also made a couple of comments on https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-6207

File: ops/rak_button_firmware.ahk (hunk 1 above)
Comment (BraveKerry) [reply ops/rak_button_firmware.ahk:156]: Correct

File: ops/rak_button_firmware.ahk (hunk 4 above)
Comment (BraveKerry) [reply ops/rak_button_firmware.ahk:95]: Correct, the script ends on screen where the status values can be checked.

File: ops/rak_button_firmware.ahk (hunk 2 above)
Comment (BraveKerry) [reply ops/rak_button_firmware.ahk:177]: I have updated the instructions now.

File: ops/rak_button_firmware.ahk (hunk 5 above)
Comment (BraveKerry) [reply ops/rak_button_firmware.ahk:70]: Correct, the script ends on screen where the status values can be checked.

File: ops/rak_button_firmware.ahk (hunk 3 above)
Comment (BraveKerry) [reply ops/rak_button_firmware.ahk:79]: The script assumes that the Google Sheets is tab open on the only instance of Google Chrome.

File: ops/rak_button_firmware.ahk (hunk 1 above)
Comment (trisapeace) [reply ops/rak_button_firmware.ahk:156]: Hmmm, actually, maybe these values are the same for both. And it just depends on which AWS window you're logged into in Chrome for it to work on dev or prod.

File: ops/rak_button_firmware.ahk (hunk 3 above)
Comment (trisapeace) [reply ops/rak_button_firmware.ahk:79]: OK, yup, I see that in the docs now.

File: ops/rak_button_firmware.ahk (hunk 5 above)
Comment (trisapeace) [reply ops/rak_button_firmware.ahk:70]: OK, thanks for doing those spot checks. In that case, are you confident enough that we can just skip the `at+get_config=lora:status` call entirely?

Comment (BraveKerry) [reply ops/rak_button_firmware.ahk:70]: This step is necessary for getting the DEVEUI

Comment (trisapeace) [reply ops/rak_button_firmware.ahk:70]: Oh! Cool. Thanks for explaining.

**Review by trisapeace — APPROVED** [review]

**Pull request merged**